etc/cvmfs/domain.d/kek.jp.conf

```diff
@@ -0,0 +1,9 @@
+# These are here so cvmfs will notice them if common.conf sets them
+CVMFS_USE_CDN="$CVMFS_USE_CDN"
+CVMFS_HTTP_PROXY="$CVMFS_HTTP_PROXY"
+CVMFS_FALLBACK_PROXY="$CVMFS_FALLBACK_PROXY"
+. ../common.conf
+
+CVMFS_SERVER_URL="http://cvmfs-stratum-one.cc.kek.jp:8000/cvmfs/@fqrn@;http://grid-cvmfs-one.desy.de:8000/cvmfs/@fqrn@;http://cvmfs.racf.bnl.gov:8000/cvmfs/@fqrn@"
```

Comment:
If this is a widely used repository, I think we ought to have two stratum 1s on each major continent instead of just one, like all of the other widely used repositories.

Reply:
The plan is to add RAL, FNAL, and IHEP.